validphys2/src/validphys/filters.py

```diff
@@ -107,12 +107,14 @@ class FilterDefaults:
     Dataclass carrying default values for filters (cuts) taking into
     account the values of ``q2min``, ``w2min`` and ``maxTau``.
     """
-    q2min: float
-    w2min: float
-    maxTau: float
+    q2min: float = None
+    w2min: float = None
+    maxTau: float = None
 
     def to_dict(self):
-        return dataclasses.asdict(self)
+        default_dict = dataclasses.asdict(self)
+        filtered_dict = {k: v for k, v in default_dict.items() if v is not None}
+        return filtered_dict
```

Comment:
Is this necessary? Before you were also returning `None` when there were no defaults.

Reply:
yes, you are right. I don't think it is necessary